packages/vscode-extension/package.json

```diff
@@ -480,7 +480,7 @@
       {
         "command": "fx-extension.addFeature",
         "title": "%teamstoolkit.commands.addFeature.title%",
-        "enablement": "!fx-extension.isSPFx && !fx-extension.isM365 && fx-entension.gaPreviewEnabled"
+        "enablement": "!fx-extension.isSPFx && !fx-extension.isM365 && fx-entension.previewFeaturesEnabled"
```

Comment:
Remove `!fx-extension.isM365` since M365 apps can also add clound resources and additional features.

Reply:
Thanks, fixed.